.travis.yml

```diff
@@ -56,6 +56,9 @@ addons:
 
 script:
   - ./sh/vsbuild.sh
+  # Enable this once we've figured out where to put the packages
+  # after they've been built
+  # - ./sh/vspackage.sh
```

Comment:
This shouldn't be part of `script` but party of `deploy` or `after_success`

Reply:
it should actually be part of `before_deploy`. Done